types/eonasdan-bootstrap-datetimepicker/index.d.ts

```diff
@@ -61,15 +61,14 @@ export interface Icons {
     close?: string | undefined;
 }
 
```

Comment:
What do you think about adding this? I think it will help others understand what each number means.

```typescript
/**
 * Sunday: 0
 * Monday: 1
 * Tuesday: 2
 * Wednesday: 3
 * Thursday: 4
 * Friday: 5
 * Saturday: 6
 */
```

Reply:
@ToastHawaii Thanks for reviewing! All else being equal, personally I prefer the current inline comments:
```ts
    | 0 /* Sunday */
    | 1 /* Monday */
    | 2 /* Tuesday */
    | 3 /* Wednesday */
    | 4 /* Thursday */
    | 5 /* Friday */
    | 6 /* Saturday */;
```
*but*, is there maybe an advantage to your suggestion that I'm overlooking?

Reply:
Yes, there is an advantage when you code with an IDE (like Visual Studio Code). When you use the type and hover over it the description is displayed. The inline commenents are not shown anywhere as far as I know, except when I look at the d.ts files.

I have made an example here:
https://www.typescriptlang.org/play?#code/PQKhFgCgAIWhlArgOwCYEMCeAuaAGKWaAWQHs0tcBGQuAFUQFMBnDHaAJlugHVHVkLNrgDM3OgAtEAJ1aVoAFm4AxaQEth0AKzd46AC4zNANlrAo+zAAdG0ACJYA8gDM+jANbQAvIWh+APvjQoAgobLDmMH7QgVTBcGQUmBG+AZzx0AxCWClRaSIZfALZySCR0TGKGZIycqXl0YFaGaoaOWWplcYZeobS4WUA3FAjkKPAwNBqyNAAtsmkAO4zzmoANowANND6LPpT+wBGyRKkAG6M0tDnl-ZOroweUM4oAMb6auTQ6AAUmg6YFxudwASgA3p1pIw+jM2MNIABfIA

For people like me, it is important to know that 0 = Sunday, because here (eg. in Switzerland) the first day of the week is Monday. :)

Reply:
Approve

Reply:
I approved. It is okay, even if I would do it another way.

Reply:
Feel free to add yourself to the owner list. :)